bridge/src/index.ts

```diff
@@ -441,6 +458,15 @@ process.on("uncaughtException", console.error);
 
     const slackChannel = new SlackChannel(slackWebClient, SLACK_CHANNEL_NAME);
     const slackMessageSender = new SlackMessageSender(slackChannel);
+    const planetIds = {
+        odin: PLANET_ODIN_ID,
+        heimdall: PLANET_HEIMDAL_ID,
+    };
+    const planetVaultAddress = {
+        heimdall: HEIMDALL_VALUT_ADDRESS,
```

Comment:
no vault address for odin?

Reply:
I didn't add odin's valult address because it would not be needed.